rebasehelper/spec_hooks/replace_old_version.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import re
+
+import six
+
+from rebasehelper.specfile import BaseSpecHook
+
+
+class ReplaceOldVersionSpecHook(BaseSpecHook):
+    """SpecHook for replacing occurrences of old version string."""
+
+    @classmethod
+    def _replace(cls, line, old, new, replace_with_macro=False):
+        """Replaces occurrences of old version on a line with new version.
+
+        Args:
+            line (str): String to replace the version in.
+            old (str): Old version string.
+            new (str): New version string.
+            replace_with_macro (bool): Whether %{version} macro should
+                be used as a substitution.
+
+        Returns:
+            str: Modified line with replaced old version strings.
+
+        """
+        if re.search(r"Version:", line):
```

Comment:
Is this check really necessary? Isn't the version already bumped in `rebase_spec_file`?